cmd/ethtest/main.go

```diff
@@ -26,12 +26,14 @@ import (
 	"strings"
 
 	"github.com/codegangsta/cli"
+	"github.com/ethereum/go-ethereum/core/vm"
 	"github.com/ethereum/go-ethereum/logger/glog"
 	"github.com/ethereum/go-ethereum/tests"
 )
 
 var (
 	continueOnError = false
+	vmTrace         bool
```

Comment:
Could you write `= false` here instead of relying on the default value? I know they're the same, but let's make it consistent with the others around it.